shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -659,14 +677,32 @@
             break;
 
           case 'touchstart':
-            this.canceled = e.touches.length > 1;
-            break;
+            this.cancelLongPressTimeout();
 
-          case 'contextmenu':
-            if (this.icon || this.canceled) {
+            if (e.touches.length > 1) {
+              this.canceled = true;
               return;
             }
 
+            if (this.inEditMode) {
+              this.touchStart.x = e.touches[0].pageX;
```

Comment:
Maybe we could save a reference to e.touches[0] and also a reference to x and y to reuse here https://github.com/mozilla-b2g/gaia/pull/27245/files#diff-18ee9a6d12712069d19e901b9ef493c2R695